src/renderer/atlas/BackendD3D.h

```diff
@@ -291,6 +297,9 @@ namespace Microsoft::Console::Render::Atlas
         // The bounding rect of _cursorRects in pixels.
         til::rect _cursorPosition;
 
+        const f32 _curlyLineHeight = 0.1f; // in `em` units.

```

Comment:
I hope that I'm not biased towards this curly line height 😅 I tried to strike a balance between waviness and the wave overlapping too much to the base of the font glyphs. 

See if the current waviness suits your (anyone) taste.